app/src/main/java/io/homeassistant/companion/android/settings/notification/views/NotificationDetailView.kt

```diff
@@ -0,0 +1,97 @@
+package io.homeassistant.companion.android.settings.notification.views
+
+import android.os.Parcelable
+import android.text.Html
+import android.widget.TextView
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.viewinterop.AndroidView
+import com.fasterxml.jackson.core.JsonParser
+import com.fasterxml.jackson.databind.ObjectMapper
+import io.homeassistant.companion.android.R
+import io.homeassistant.companion.android.database.notification.NotificationItem
+import io.homeassistant.companion.android.util.notificationItem
+import kotlinx.android.parcel.Parcelize
+import java.util.Calendar
+import java.util.GregorianCalendar
+
+@Parcelize
+data class NotificationData(val received: Long, val message: String, val data: String) : Parcelable
+
+@Composable
+fun LoadNotification(notification: NotificationItem) {
+    val notificationItem = rememberSaveable {
+        NotificationData(notification.received, notification.message, notification.data)
+    }

```

Comment:
So I'm not sure we need the `rememberSaveable` here?

Reply:
`rememberSaveable` will retain when the UI is redrawn like when the orientation changes.

https://developer.android.com/jetpack/compose/state#restore-ui-state

I think I added this in the first PR we made to pass in the state?  I can remove it if you like

Reply:
You only need the `remember` and `rememberSavable` if the state only exists inside the compose function.  Since the data is coming from outside the compose function we don't need to save it there.